source/new.rst

```diff
@@ -167,9 +167,13 @@ Alice and Bob.
      which Bob's device uses in step 4
      to authenticate itself against Alice's device.
 
+   - optionally up to 64 bytes of metadata
+     such as ``INVITE-TO=groupname``
```

Comment:
@r10s what's the maxsize of a delta-group name? (internally it's an integer but we are talking about the humand-readable group name).  I suspect it's larger than 64 bytes.